test/steps_test.go

```diff
@@ -329,6 +331,8 @@ func FeatureContext(s *godog.Suite) {
 	s.Step(`^I get the method from the Interface by name "([^"]*)"$`, iGetTheMethodFromTheInterfaceByName)
 	s.Step(`^I get the method from the Contract by name "([^"]*)"$`, iGetTheMethodFromTheContractByName)
 	s.Step(`^the produced method signature should equal "([^"]*)"\. If there is an error it begins with "([^"]*)"$`, theProducedMethodSignatureShouldEqualIfThereIsAnErrorItBeginsWith)
+	s.Step(`^a source map json file "([^"]*)"$`, aSourceMapJsonFile)
+	s.Step(`^the string composed of pc:line number equals "([^"]*)"$`, theStringComposedOfPclineNumberEquals)
```

Comment:
Is there a reason why these tests are in @unit and use a mock server response? Seems more robust to have an actual server call in @integration in case the format of the response object changes.

Reply:
A few thoughts here:
* I'm on-board with an integration test primarily as a means to vet all SDKs can request a sourcemap during compilation.
* I think the existing unit test adds a valuable proof point and I'd optionally favor retaining it.  
* I agree there's value in defending against response changes.  However, since we're discussing an external protocol (sourcemap) that's remained stable for _years_,  I think the chance of response format changes are _low_.
  * Aside:  In future iterations where we also support PyTeal-to-TEAL mapping, I envision supporting multiple sources.  We'll need a richer SDK model + more tests.

Reply:
I can get behind that reasoning. Resolving.